Markdown lint is too greedy!

### Describe the bug

I ran `make all` and was overrun with markdownlint warnings.

### How to Reproduce

Do some development on the UI, which populates `./ui/node_modules` and then the lint will never pass.

### Expected Behavior

The lint should only check docs that are under our control (i.e not paths that are .gitignored).

### Deployment Environment

_No response_

### Additional Context

_No response_

to quickly reproduce
```bash
mkdir -p ui/node_modules
echo "[@]()" > ui/node_modules/foo.md
make mardown-lint
```

outputs:
```
ui/node_modules/foo.md:1:1 MD042/no-empty-links No empty links [Context: "[@]()"]
make: *** [Makefile:83: markdown-lint] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -77,7 +77,7 @@ yaml-lint: ## Lint the yaml files
 .PHONY: markdown-lint
 markdown-lint: ## Lint markdown files
 	$(ECHO_PREFIX) printf "  %-12s ./...\n" "[MD LINT]"
-	$(CMD_PREFIX) docker run -v $(CURDIR):/workdir docker.io/davidanson/markdownlint-cli2:v0.6.0 "**/*.md" "#node_modules" > /dev/null
+	$(CMD_PREFIX) docker run -v $(CURDIR):/workdir docker.io/davidanson/markdownlint-cli2:v0.6.0 "**/*.md" "#ui/node_modules" > /dev/null
 
 .PHONY: gen-docs
 gen-docs: ## Generate API docs

